script.js

```diff
@@ -1 +1,51 @@
-console.log("hello world")
\ No newline at end of file
+let green = "rgb(0, 255, 0)";
+let red = "rgb(255, 0, 0)";
+
+document.querySelectorAll(".klasa").forEach(klasa => {
```

Comment:
This approach is okay, but in the future try using more of 
https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator

and also in CSS file so 

```suggestion
document.querySelectorAll("#klasa > div").forEach(klasa => {
```

and on your parent div you add `id=klasa`